### PR TITLE
Add RBAC note to library panels

### PR DESCRIPTION
**Pull request by natellium**
**What is this feature?**

Adds a note to the library panels docs about RBAC. See https://github.com/grafana/grafana/pull/73475

### CODE REVIEW

**Review by imatwawana — APPROVED** [review]
I've made a couple formatting fixes, but otherwise this LGTM!

**Comment by natellium** [comment]
@imatwawana did you commit the fixes? I don't see changes.

**Comment by imatwawana** [comment]
> @imatwawana did you commit the fixes? I don't see changes.

Weird, I don't know what happened there. Let me add the suggestions again.

File: docs/sources/dashboards/build-dashboards/manage-library-panels/index.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ A library panel is a reusable panel that you can use in any dashboard. When you
 
 You can save a library panel in a folder alongside saved dashboards.
 
+**Note:** When [RBAC](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/) is enabled, you need to have the relevant [Permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/) to create, manage, and view library panels.
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/manage-library-panels/index.md:27]:
```suggestion
{{% admonition type="note" %}}
When [RBAC](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/) is enabled, you need to have the relevant [Permissions](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/) to create, manage, and view library panels.
{{% /admonition %}}
```

**Review by imatwawana — COMMENTED** [review]
Hopefully it worked this time?

File: docs/sources/dashboards/build-dashboards/manage-library-panels/index.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/dashboards/build-dashboards/manage-library-panels/index.md:27]: FYI: I'm using version interpolation syntax in the links which ensures that it always points to the right place.

**Comment by natellium** [comment]
@imatwawana yep, got it now!

**Pull request merged**